gen/aircraft.py

```diff
@@ -389,7 +389,7 @@ def setup_group_activation_trigger(self, flight, group):
 
     def generate_planned_flight(self, cp, country, flight:Flight):
         try:
-            if flight.client_count == 0 and self.game.perf_ai_parking_start:
```

Comment:
The perf_ai_parking_start variable is placed beneath "game.settings" and not directly under "game."